stdlib/log-api/src/main/java/org/ballerinalang/stdlib/log/Utils.java

```diff
@@ -103,4 +112,36 @@ public static void printWarn(Object msg) {
                     });
         }
     }
+
+    public static void setModuleLogLevel(BString logLevel, Object moduleName) {
+        String module;
+        if (moduleName == null) {
+            module = getPackagePath();
+        } else {
+            module = moduleName.toString();
+        }
+        String level = logLevel.getValue();
+        switch (level) {
+            case debug:
+                LOG_MANAGER.setModuleLogLevel(BLogLevel.DEBUG, module);
+                break;
+            case error:
+                LOG_MANAGER.setModuleLogLevel(BLogLevel.ERROR, module);
+                break;
+            case info:
+                LOG_MANAGER.setModuleLogLevel(BLogLevel.INFO, module);
+                break;
+            case warn:
+                LOG_MANAGER.setModuleLogLevel(BLogLevel.WARN, module);
+                break;
+            case trace:
+                LOG_MANAGER.setModuleLogLevel(BLogLevel.TRACE, module);
+                break;
+            case off:
+                LOG_MANAGER.setModuleLogLevel(BLogLevel.OFF, module);
+                break;
+            default:
+                LOG_MANAGER.setModuleLogLevel(BLogLevel.ALL, module);
```

Comment:
Instead of this, you can use the existing API provided for converting a string log level value to a `BLogLevel` enum. 
```java
LOG_MANAGER.setModuleLogLevel(BLogLevel.toBLogLevel(level), module);
```

Reply:
Updated in a15bfd2